php/context/class-fieldmanager-context-quickedit.php

```diff
@@ -134,19 +157,20 @@ public function add_quickedit_box( $column_name, $post_type, $values = array() )
 	/**
 	 * Callback for wp_ajax_fm_quickedit_render.
 	 * Renders a form with pre-filled values to replace the one generated by $this->add_quickedit_box().
-	 * @return string
+	 *
+	 * @return void
```

Comment:
Per the docs standards, `return @void` shouldn't be used outside of bundled themes: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods.